types/mongoose-paginate-v2/index.d.ts

```diff
@@ -30,8 +30,7 @@ declare module 'mongoose' {
     interface PaginateOptions {
         /* tslint:disable-next-line: ban-types */
         select?: Object | string;
```

Comment:
You might want to fix this one too.

Reply:
> In my defence, I did suggest `string | object` as an alternative.

Yeah sorry I did not want that to sound rude. I will simply call it learning.

> You might want to fix this one too.

Done!

Reply:
Also, I removed additional uses of Object, which were there for the same reason. There are no more `/* tslint:disable-next-line: ban-types */` (or any exception) in the two packages.

Reply:
No harm done - I shouldn't have been snippy. :smile: